src/Components/PaymentFilter/PayoneBNPLPaymentMethodFilter.php

```diff
@@ -4,21 +4,51 @@
 
 namespace PayonePayment\Components\PaymentFilter;
 
+use PayonePayment\Components\ConfigReader\ConfigReader;
 use PayonePayment\Core\Utils\AddressCompare;
 use Shopware\Core\Checkout\Customer\Aggregate\CustomerAddress\CustomerAddressEntity;
 use Shopware\Core\Checkout\Order\Aggregate\OrderAddress\OrderAddressEntity;
 use Shopware\Core\Checkout\Payment\PaymentMethodCollection;
+use Shopware\Core\System\SystemConfig\SystemConfigService;
 
 class PayoneBNPLPaymentMethodFilter extends DefaultPaymentFilterService
 {
+    public function __construct(
```

Comment:
this constructor does not take any effect.
Please remove it.

or do i miss something?

Reply:
Yes, the SystemConfigService is injected specifically for this class, so the Constructor is required here.